README.md

```diff
@@ -134,15 +136,16 @@ This will be a simple example that will add the test suite `rpi` with the `RpiTe
 }
 ```
 This will add a test suite `rpi` that will include the test `RpiTest.via` and all of the tests included in the `common` test suite.
+
 3. Try it out to verify it all works and your tests pass:
-```shell
+```console
 $ node ./test.js -n -t rpi
 ```
 
 ## Running the HTTP test server
 
 ### Overview
-Vireo utilizes [httpbin](https://httpbin.org/) for testing the HTTP Client functionality. The tests rely on a locally running instance of the httpbin server. The test suite will skip tests (without failing) that rely on the httpbin server running locally if the server is not running. However, if you would like to run the HTTP client tests locally these instructions will help you get configured.
+Vireo uses [httpbin](https://httpbin.org/) for testing the HTTP Client functionality. If the httpbin server is not running locally, the test suite will skip those tests that rely on it. If you would like to run the HTTP client tests locally these instructions show you how to do so.
```

Comment:
It's more accurate to say Vireo HTTP tests rely on a locally running instance of the [httpbin](https://github.com/kennethreitz/httpbin) server. Just saying we use httpbin might not be clear enough.